### PR TITLE
added one-up titles

### PR DESCRIPTION
**Pull request by sjmog**
With title, title + leader + subtitle, and subtitle-only modules

### CODE REVIEW

**Comment by sjmog** [comment]
Paging @pitchinvasion for a review

**Comment by sjmog** [comment]
Closes #76 .

**Comment by sjmog** [comment]
(Also added it to the `index` page)

File: source/index.erb [hunk 1]
```diff
@@ -38,7 +38,7 @@ title: MA Style Guide
     </section>
     <section class="holler-block__section">
       <div class="cta-block">
-        <%= link_to "See it in action", "homepage", class: "cta cta--horizontal" %>
+        <%= link_to "See it in action", "homepage.html", class: "cta cta--horizontal" %>
```
Comment (pitchinvasion) [inline source/index.erb]: that's weird you shouldn't need the .html here, middleman adds this

**Comment by pitchinvasion** [comment]
a bit late but :+1:

**Pull request merged**